conftest.py

```diff
@@ -256,15 +256,19 @@ def reset_tensix(request, silicon_arch_name):
 
 
 @pytest.fixture(scope="function")
-def device_init_destroy(request):
+def device_l1_small_size(request):
```

Comment:
I discussed the naming with @davorchap, I originally called it `L1_RESERVED` but it was confusing because there's already `L1_UNRESERVED`.  Another name I thought of was `L1_STATIC`.  I'm not tied to the name small, we just needed a name that doesn't collide with an existing concept.  That said, this region is intended for small allocations, so as to not have them interleaved with bigger ones and causing fragmentation problems.